app/views/summaries/index.html.erb

```diff
@@ -1,8 +1,10 @@
+<h1 class="title header-centered ">Total Spending</h1>
+<%= pie_chart @totals, donut: true, legend: false  %>
 <% if Purchase.any? %>
-  <p>
+  <p class="line-spacing-top">
     <%= "Total spending to date: " + number_to_currency(Purchase.where(date: @date_start..@date_end).to_a.map(&:amount).sum) %>
```

Comment:
Just FYI, since the table is leaving, but rather than writing things like `Purchase.where(date: @date_start..@date_end).to_a.map(&:amount).sum` inline, it'd be better to assign them to an instance variable